src/wallet.rs

```diff
@@ -236,7 +236,7 @@ pub(crate) fn get_esplora_url(network: Network, user_provided_url: Option<String
         match network {
             Network::Bitcoin => "https://blockstream.info/api",
             Network::Testnet => "https://blockstream.info/testnet/api",
-            Network::Signet => "https://mempool.space/signet/api",
+            Network::Signet => "https://mutinynet.com/api",
```

Comment:
Starting to get into config hell but hopefully once everything is switched over to signet we can not worry about testnet as much.